.cirrus.yml

```diff
@@ -545,28 +579,19 @@ task:
 
   ccache_cache:
     folder: $CCACHE_DIR
-  configure_script: |
```

Comment:
I assume you did this because the pkg-config test fails without PKG_CONFIG_PATH set? How about passing get_option('pkg_config_path') to the test instead? I think that should also work and be more reliable.